test/src/test/java/jenkins/install/LoadDetachedPluginsTest.java

```diff
@@ -81,6 +84,25 @@ public void upgradeFromJenkins2() {
         });
     }
 
+    @Issue("JENKINS-48604")
+    @Test
+    @LocalData
+    public void upgradeFromJenkins2WithDependency() {
+        VersionNumber since = new VersionNumber("2.0");
+        rr.then((JenkinsRule r) -> {
```

Comment:
```java
rr.then(r -> {
```

suffices I think.

Reply:
Yeah I think my IDE auto-corrected from `r -> {` when I copied an existing test. I'll clean it up.